scanner/src/main/kotlin/scanners/fossid/FossIdScanResults.kt

```diff
@@ -124,10 +125,11 @@ internal fun mapSnippetFindings(
                     if (expression == null) {
                         issues += FossId.createAndLogIssue(
                             source = "FossId",
-                            message = "Failed to map license '$it' as an SPDX expression."
+                            message = "Failed to map license '$it' as a SPDX expression. Mapping it using licenseRef-.",
+                            severity = Severity.HINT
                         )
                     }
-                }
+                } ?: "LicenseRef-fossid-${it.replace("[^a-zA-Z0-9\\-.]".toRegex(), "-")}".toSpdx()
```

Comment:
In the analyzer we have `unmappedDeclaredLicenses` field in the model and only a warning (or hint?) if the license could not be mapped. This allows to write a (tweaked to users needs) policy rule to flag the mapping issue, under desired severity under desired preconditions. See, `rules.kts` in `ort-config` repository. 

How about doing the same here?

Reply:
Answered below.